source/latest_version_spirv_header.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LIBSPIRV_LATEST_VERSION_SPIRV_HEADER_H_
+#define LIBSPIRV_LATEST_VERSION_SPIRV_HEADER_H_
+
+#include "spirv/1.2/spirv.h"
+
+inline const char* GetSpirvHeaderVersionString() {
+  return "1.2";
```

Comment:
This function is not used.

Reply:
spirv.h has a self-reported version number via SPV_VERSION and SPV_REVISION macros, and static variables SpvVersion and SpvRevision.